data/maps/landmarks.asm

```diff
@@ -4,7 +4,7 @@ landmark: MACRO
 ENDM
 
 Landmarks:
-	landmark   0,   0, SpecialMapName
+	dbbw       0,   0, SpecialMapName
```

Comment:
If the `landmark` macro hasn't been modified to shift its coordinates by 8/16, then `dbbw` isn't necessary here.

Reply:
(Updating the macro and all the coords would be nice, to match the Tilemap Studio status bar, but rust reverting this would be OK.)

Reply:
This is still `dbbw`.